### PR TITLE
Maint fauna changes

### PR DESCRIPTION
**Pull request by Alliostra**
## About The Pull Request

Spiders:

Nurse spiders now have a 4% chance per attack to inject someone with spider eggs, which will hatch into 4 to 8 spiderlings after some time. This is cool.

Pararein (used by hunters and warriors) now does much less sanity damage, but it decreases your toughness and vigilance, making it somewhat debiliating. It also has a slight insight gain effect and buffs your cog (for the memes)

Aranecolmin is now a proper toxin that is used by nurse spiders. It is an average strength (read: very weak) toxin which grants you a small painkiller effect and speed up the metabolism of pararein greatly.

The idea here is that not running from spiders, but fighting them makes you able to fight on better (because the weakening pararein which you got from the fast hunters attacking you is quickly metabolized), but it will actually deal some toxin damage while keeping you in the belief that your status is okay thanks to painkillers, lulling you into a false sense of strength. 

This also makes the spider toxins useful as ghetto painkillers/drugs.

Nurse spider was made slightly faster and made it inject a bit more toxin per attack
Hunter spider injects more toxins per attack

removed a few spider-related files and moved their contents to the main spider file

added some names and descriptions to maint fauna meat and shuffled the toxins in spider meat to reflect the toxins that the respective spider uses.

Roaches:
buffed the average damage of most roaches by one point, most notably the kinds that currently can't even hurt you if you have 30% melee resist on all body parts.
Nerfed the roachling though, it was better than the kampfer before :yis:

I may have forgotten something, it's late.
## Why It's Bad For The Game

Balance changes and stealth carrion buffs by a non-player no less, weeeeee

but seriously, maint fauna is not dangerous enough if since you can easily outtank anything short of a fuhrer or a carrion nest. I hope the changes to spiders won't buff carrions too much, but since monkey organs no longer give gene points I'm hoping this doesn't evolve into a problem.

## Changelog
:cl:
tweak: made maint fauna meat more descriptive.
balance: rebalanced spiders and buffed the melee damage of most roaches slightly
add: nurse spiders can inject people with spider eggs sometimes
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/superior_animal/giant_spider/types/nurse.dm [hunk 1]
```diff
@@ -13,23 +13,28 @@
...
 
 /mob/living/carbon/superior_animal/giant_spider/nurse/attemptAttackOnTarget()
-	var/target = ..()
-	if(ishuman(target))
-		var/mob/living/carbon/human/H = target
-		if(prob(poison_per_bite))
+	..()
+	if(ishuman(target_mob))
+		var/mob/living/carbon/human/H = target_mob
+		if(prob(egg_inject_chance))
```
[the remark this answers is not in the document]
Comment (Alliostra) [reply code/modules/mob/living/carbon/superior_animal/giant_spider/types/nurse.dm:30]: wasp_lay_egss_kinda_liek_this

[the remark this answers is not in the document]
Comment (TheShown911) [reply code/modules/mob/living/carbon/superior_animal/giant_spider/types/nurse.dm:30]: Spiders don't have stingers but okay.

**Comment by TheShown911** [comment]
Solve linter failure.

**Comment by drexample** [comment]
Linter moment.
If not fixed will be closed next thursday-friday.

File: code/modules/mob/living/carbon/superior_animal/giant_spider/giant_spider.dm [hunk 2]
```diff
@@ -39,7 +39,14 @@
...
+/mob/living/carbon/superior_animal/giant_spider/New(var/location, var/atom/parent)
 	get_light_and_color(parent)
+	..()
+
+/mob/living/carbon/superior_animal/giant_spider/UnarmedAttack(atom/A, proximity)
+	. = ..()
+
+	var/mob/living/L = A
+	if(istype(L) && L.reagents)
+			L.reagents.add_reagent(poison_type, poison_per_bite)
```
Comment (TheShown911) [inline code/modules/mob/living/carbon/superior_animal/giant_spider/giant_spider.dm:51]:
```suggestion
	if(istype(L) && L.reagents)
		L.reagents.add_reagent(poison_type, poison_per_bite)
```

Comment (TheShown911) [reply code/modules/mob/living/carbon/superior_animal/giant_spider/giant_spider.dm:51]: Assume direct control

**Review by TheShown911 — APPROVED** [review]

**Comment by TheShown911** [comment]
New code is untested.

**Pull request merged**